### PR TITLE
Doc changes to prevent conflicts due to including rebuilt client build

### LINKED ISSUE #715: Contributing: prevent merge conflicts
Currently, if a PR with changes in the client is open for a while, it is almost guaranteed to cause git conflicts. We could prevent those by advising contributors to open those PRs as WIP and not including a built client, until the PR is ready to merge and approved.

### PR DESCRIPTION
**Pull request by AnandBaburajan**
Fixes #715 

## Contributor Checklist:

* [x] I have added or updated my entry in [the creators.json file](https://github.com/LiberTEM/LiberTEM/blob/master/packaging/creators.json)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -4,6 +4,7 @@
 * [ ] I have added [a changelog entry](https://github.com/LiberTEM/LiberTEM/tree/master/docs/source/changelog) for my contribution
 * [ ] I have added/updated documentation for all user-facing changes
 * [ ] I have added/updated test cases
+* [ ] I have included the rebuilt [production build of the client](https://libertem.github.io/LiberTEM/contributing.html?#building-the-client)
```
Comment (sk1p) [inline .github/PULL_REQUEST_TEMPLATE:7]: Possibly should mention that this is only needed if the GUI code actually changed - we can't rely on people actually reading the documentation link, sadly :grin:

File: docs/source/contributing.rst [hunk 2]
```diff
@@ -314,8 +314,9 @@ You can then use any editor you like to change the client source files, in the c
 We recommend `Visual Studio Code <https://code.visualstudio.com/>`_ for its excellent TypeScript support.
 
 To simplify development and installing from a git checkout, we currently always ship a production build
-of the client in the git repository. When you are creating a pull request for the client, please always
-include a current production build. You can create it using a tox shortcut:
+of the client in the git repository. Please always open your pull request for the client as WIP and
+include a rebuilt production build after the PR is approved and ready to merge.
+You can create it using a tox shortcut:
```
Comment (sk1p) [inline docs/source/contributing.rst:319]: :+1:

**Comment by uellue** [comment]
Don't worry about the CI failure. It is unrelated to this change. Being worked on in #713 and #719!

**Comment by uellue** [comment]
Thank you, well done, merged! :-)

**Pull request merged**